### PR TITLE
[Doc] add debugging tips

### PR DESCRIPTION
**Pull request by youkaichao**
Add common tips for debugging hang/crash.

### CODE REVIEW

**Review by zhuohan123 — APPROVED** [review]

File: docs/source/getting_started/debugging.rst [hunk 1]
```diff
@@ -0,0 +1,34 @@
+.. _debugging:
+
+Debugging Tips
+===============
+
+Debugging hang/crash issues
+-----------------------------
```
Comment (rkooo567) [inline docs/source/getting_started/debugging.rst:7]:
```suggestion
---------------------------
```

I think normal practice is the line length matches the sentence

File: docs/source/getting_started/debugging.rst [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+When an vLLM instance hangs or crashes, it is very difficult to debug the issue. Here are some tips to help debug the issue:
+
+- Set the environment variable `export VLLM_LOGGING_LEVEL=DEBUG` to turn on more logging.
+- Set the environment variable `export NCCL_DEBUG=TRACE` to turn on more logging for NCCL.
+- Set the environment variable `export VLLM_TRACE_FUNCTION=1` . All the function calls in vLLM will be recorded. Inspect these log files, and tell which function crashes or hangs. **Note: it will generate a lot of logs and slow down the system. Only use it for debugging purposes.**
```
Comment (rkooo567) [inline docs/source/getting_started/debugging.rst:13]: Also mention CUDA_LAUNCH_BLOCKING?

**Review by rkooo567 — APPROVED** [review]
nit comments

File: docs/source/getting_started/debugging.rst (hunk 1 above)
Comment (youkaichao) [reply docs/source/getting_started/debugging.rst]: fixed in [cb4d7ff](https://github.com/vllm-project/vllm/pull/5409/commits/cb4d7fff82d1bff7de12121bf96a04e4705895d0)

File: docs/source/getting_started/debugging.rst (hunk 2 above)
Comment (youkaichao) [reply docs/source/getting_started/debugging.rst]: fixed in [cb4d7ff](https://github.com/vllm-project/vllm/pull/5409/commits/cb4d7fff82d1bff7de12121bf96a04e4705895d0)

**Comment by DarkLight1337** [comment]
Some users have also reported slow loading of models (#5062 #5365 #5405). A troubleshooting guide for this issue would also be helpful.

**Comment by youkaichao** [comment]
@DarkLight1337 do you mean the "hang" may be caused by long model loading time?

**Comment by DarkLight1337** [comment]
> @DarkLight1337 do you mean the "hang" may be caused by long model loading time?

It seems like it. They are all hanging after the logger prints the line
```
vLLM is using nccl==<version>
```

**Comment by youkaichao** [comment]
@DarkLight1337 added in https://github.com/vllm-project/vllm/pull/5430 .

**Pull request merged**